resources/rest-service/cloudify/install_agent.py

```diff
@@ -159,6 +181,19 @@ def install(self):
         finally:
             self.runner.rm_dir(path)
 
+    def _get_certificate_file(self, path=None):
+        if not self.cloudify_agent.get('verify_rest_certificate'):
+            return None
+
+        # we were told to verify the cert, but didn't get the cert to verify
```

Comment:
maybe im misunderstanding this, but i think the comment should be inside the if, not above it